src/inmanta/deploy/state.py

```diff
@@ -54,7 +54,7 @@ def provides_view(self) -> Mapping["ResourceIdStr", Set["ResourceIdStr"]]:
         return self.reverse_mapping()
 
 
-class ComplianceStatus(StrEnum):
+class Compliance(StrEnum):
```

Comment:
rename 3.

motivation: reduce verbosity.

notes:
- not required for any name conflicts, can be dropped if not considered an improvement.
- rename of `ResourceState.status` -> `ResourceState.compliance` will require a db migration script.

Reply:
I like it.